doc/whats_new.rst

```diff
@@ -65,6 +65,8 @@ Changelog
 
     - Big rewrite of simulation module by `Yousra Bekhti`_, `Mark Wronkiewicz`_, `Eric Larson`_ and `Alex Gramfort`_. Allows to simulate raw with artefacts (ECG, EOG) and evoked data, exploiting the forward solution. See :func:`mne.simulation.simulate_raw`, :func:`mne.simulation.simulate_evoked` and :func:`mne.simulation.simulate_sparse_stc`
 
+    - Add support for drawing scalp maps by selecting an area in :func:`mne.Evoked.plot` by `Jaakko Leppakangas`_
```

Comment:
I'll try tonight.

yesterday my mac anaconda setup got me errors. I'll keep you posted.